tools/convert/README.md

```diff
@@ -0,0 +1,40 @@
+# Configuration Conversion Tool
+
+## Summary
+
+Refinery has been around long enough that its configuration files have accumulated some cruft. For backwards-compatibility reasons, we've had to leave defaults unchanged as we adjusted refinery's operations. It now works better than it did, but by default can't take advantage of new things.
+
+The `convert` tool is here to make it possible to upgrade Refinery by converting the old configuration to a new form, while preserving the important settings.
+
+It also allows all of the comments in the configuration to be rewritten for clarity and accuracy.
+
+## Usage
+
+```
+Usage:
+  convert [OPTIONS]
+
+        This tool converts a Refinery v1 config file (usually in TOML, but JSON and YAML are also
+        supported) to a Refinery v2 config file in YAML. It reads the v1 config file, and then writes
+        the v2 config file, copying non-default values from their v1 location to their v2 location
+        (if they still apply). The new v2 config file is commented in detail to help explain what
+        each value does in the new configuration.
+
+        For example, if the v1 file specified "MetricsAPIKey" in the "HoneycombMetrics" section, the v2
+        file will list that key under the "LegacyMetrics" section under the "APIKey" name.
+
+        By default, it reads config.toml and writes to stdout. It will try to determine the
+        filetype of the input file based on the extension, but you can override that with
+        the --type flag.
+
+
+Application Options:
+  -i, --input=       the Refinery v1 config file to read (default: config.toml)
+  -o, --output=      the Refinery v2 config file to write (default: -)
```

Comment:
what does `default: -` mean in this context?

Reply:
It's a convention meaning stdout (since defaulting to "stdout" would send it to a file of that name). I've used a lot of tools that do it that way...but maybe it's not conventional enough?

The other way to do this would be to set no default value and detect "" in the code instead of "-".

Reply:
I changed the default to empty string and updated the comment and the doc.